libraries/tickv/SPEC.md

```diff
@@ -175,15 +175,22 @@ If all the objects in a region are no longer valid then that region will be
 erased when `garbage_collect()` is called. Note that even if the flash is
 full `garbage_collect()` will not be called automatically.
 
-### Zeroizing keys
+### Zeroising keys
```

Comment:
I see you trying to sneak in the King's spelling!

Reply:
> I see you trying to sneak in the King's spelling!

I'm not sure how I feel about this...

Reply:
It matches the Wikipedia article and the rest of TicKV